app/views/materials/_material.html.erb

```diff
@@ -1,10 +1,11 @@
-<%= render "content/teaser", content: material do %>
```

Comment:
I updated this template in #181. Basically just wrap the innards in `<%= render "content/teaser", content: material do %>...<% end %>` instead of `<div class="material card">...</div>`.